Modules/Project/Resources/assets/js/app.js

```diff
@@ -2,4 +2,14 @@ $(document).ready(function () {
 	$("input").on("change", function () {
 		this.value = (this.value).replace(/\s+/g, " ");
 	});
+
+	$(".form-body").on("keyup", ".daily-effort",function(){
+	var daily_effort=$(this).val();
```

Comment:
```suggestion
	var dailyEffort=$(this).val();
```
Please make changes at other places as well.